docs/cloud/integrations/grafana.md

```diff
@@ -0,0 +1,1011 @@
+            "value": "ap-east-1"
+          },
+          {
+            "selected": false,
+            "text": "ap-south-1",
+            "value": "ap-south-1"
+          },
+          {
+            "selected": false,
+            "text": "ap-northeast-3",
+            "value": "ap-northeast-3"
+          },
+          {
+            "selected": false,
+            "text": "ap-northeast-2",
+            "value": "ap-northeast-2"
+          },
+          {
+            "selected": false,
+            "text": "ap-southeast-1",
+            "value": "ap-southeast-1"
+          },
+          {
+            "selected": false,
+            "text": "ap-southeast-2",
+            "value": "ap-southeast-2"
+          },
+          {
+            "selected": false,
+            "text": "ap-northeast-1",
+            "value": "ap-northeast-1"
+          },
+          {
+            "selected": false,
+            "text": "ca-central-1",
+            "value": "ca-central-1"
+          },
+          {
+            "selected": false,
+            "text": "eu-central-1",
+            "value": "eu-central-1"
+          },
+          {
+            "selected": false,
+            "text": "eu-west-1",
+            "value": "eu-west-1"
+          },
+          {
+            "selected": false,
+            "text": "eu-west-2",
+            "value": "eu-west-2"
+          },
+          {
+            "selected": false,
+            "text": "eu-south-1",
+            "value": "eu-south-1"
+          },
+          {
+            "selected": false,
+            "text": "eu-west-3",
+            "value": "eu-west-3"
+          },
+          {
+            "selected": false,
+            "text": "eu-north-1",
+            "value": "eu-north-1"
+          },
+          {
+            "selected": false,
+            "text": "me-south-1",
+            "value": "me-south-1"
+          },
+          {
+            "selected": false,
+            "text": "sa-east-1",
+            "value": "sa-east-1"
+          }
+        ],
+        "query": "us-east-2,us-east-1,us-west-1,us-west-2,af-south-1,ap-east-1,ap-south-1,ap-northeast-3,ap-northeast-2,ap-southeast-1,ap-southeast-2,ap-northeast-1,ca-central-1,eu-central-1,eu-west-1,eu-west-2,eu-south-1,eu-west-3,eu-north-1,me-south-1,sa-east-1",
+        "queryValue": "",
+        "skipUrlSync": false,
+        "type": "custom"
+      }
+    ]
+  },
+  "time": {
+    "from": "now-6h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "AWS S3",
+  "uid": "aws_s3_json",
+  "version": 1,
+  "weekStart": ""
+}
+```
+</details>
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/grafana-aws-s3-dashboard.png" />
+</div>
+
+## Send alerts
+
+Suppose you'd like to be notified when the CPU utilization crosses a threshold value. You can use Grafana's `Alerting` feature to create an `Alert Rule` with this query and expression to evaluate the threshold value.
```

Comment:
```suggestion
Suppose you'd like to be notified when the CPU utilization crosses a threshold value. You can use Grafana's `Alerting` feature to create an `Alert Rule`. 
```

Reply:
This needs a bit more explanation. I guess $__timeFilter is the name for the expression that's built in the editor at the bottom of the screen?

Reply:
I've updated the text and the images to suit the text. 

Does this change look good to you?